pkg/validation/projects.go

```diff
@@ -82,11 +69,27 @@ func ValidateProjects(projects []v1alpha2.Project) (returnedErr error) {
 }
 
 // validateRemoteMap checks if the checkout remote is present in the project remote map
-func validateRemoteMap(remotes map[string]string, checkoutRemote, projectName string) error {
+func validateRemoteMap(remotes map[string]string, checkoutRemote, object, name string) error {
```

Comment:
I think objectType or objectKind would be clearer
```suggestion
func validateRemoteMap(remotes map[string]string, checkoutRemote, objectType, ObjectName string) error {
```

or even componentType, componentName... I know it's probably too long but really straightforward.

Reply:
updated to `objectType` and `objectName`